styles.css

```diff
@@ -1,155 +1,103 @@
-.navgation {
-  display: flex;
-  flex-wrap: nowrap;
-  flex-grow: 1;
-  max-width: 100vw;
-  max-height: 15vw;
-}
-.menu {
-  max-width: 10vw; 
-  max-height: 5vw;
-  margin-left: 73vw;
-  margin-right: 0vw;
-}
-.para {
-  width: 100vw;
-  height: 33vw;
-  font-family: 'Poppins';
-  font-style: normal;
-  font-weight: 400;
-  font-size: 5vw;
-  line-height: 6vw;
-  letter-spacing: 0.4vw;
-  color: #344563;
-  flex: none;
-  order: 1;
-  flex-grow: 0;
-}
-.feyisa {
-  width: 100vw;
-  height: 40vm;
-  font-family: 'Poppins';
-  font-style: normal;
-  font-weight: 700;
-  font-size: 15vw;
-  line-height: 13vw;
-  display: flex;
-  align-items: center;
-  letter-spacing: 0.1vw;
-  color: #172B4D;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  top: 25%;
-}
-.let {
-  width: 100vw;
-  height: 10vw;
-  font-family: 'Poppins';
-  font-style: normal;
-  font-weight: 500;
-  font-size: 3vw;
-  line-height: 2vw;
-  color: #7F8CFF;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-}
-.icon{
-display: flex;
-flex-grow: 1;
-max-width: 100vw;
-}
-.icon-1{
-  display: flex;
-  flex-grow: 1;
-  max-width: 100vw;
-  }
-.logo {
-  max-width: 40vw;
-  font-family: 'Poppins';
-  font-style: normal;
-  font-weight: 700;
-  font-size: 5vw;
-  line-height: 5vw;
-  display: flex;
-  align-items: start;
-  text-align: start;
-  color: #6070FF;
-  flex: none;
-  order: 0;
-}
-.click{
-width: 100vw;
-height: 20vw;
-flex-flow: row nowrap;
-justify-content: space-around;
-margin-left: 1vw;
-margin-top:  2vw;
-flex-grow: 1;
-}
-.click-1{
-  margin-left: 1vw;
-  margin-top:  10vw;
-}
-.top{
-display: flex;
-flex-grow: 1;
-max-width: 100vw;
-max-height: 10vw;
-}
-.top-1{
-  display: flex;
-  flex-grow: 1;
-  max-width: 100vw;
-  max-height: 30vw;
-  }
-.inner{
-margin-left: 1vw;
-margin-top:  1vw;
-}
-.bottom{
-  max-width: 10vw;
-  margin-left: 50vw;
-  margin-top: 0vw;
-}
-.bottom-3{
-  margin-left: 15vw;
-}
-.bottom-2{
-  margin-left: 30vw;
-}
-.top-el{
-  margin-left: 30vw;
-}
-.cricle{
-  margin-left: 50vw;
-}
-.top-elip{
-  margin-left: 40vw;
-}
-.twitter{
-  max-width: 3vw;
-  height: auto;
-  margin-left: 0vw;
-}
-.in{
-  max-width: 4vw;
-  height: auto;
-  margin-left: 4vw;
-}
-.medium{
-  max-width: 4vw;
-  height: auto;
-  margin-left: 4vw;
-}
-.github{
-  max-width: 4vw;
-  height: auto;
-  margin-left: 4vw;
-}
-.angellist{
-  max-width: 4vw;
-  height: auto;
-  margin-left: 4vw;
-}
+main{
+  width: 100vw;
+  min-height: 100vh;
+  background-image: url("background.png");
+  background-size: 105vw 23vw;
+  background-repeat: no-repeat;
+}
+.navgation {
+  display: flex;
+  flex-wrap: nowrap;
+  flex-grow: 1;
+  max-width: 100vw;
+  max-height: 15vw;
+  align-items: center;
+}

```

Comment:
- [ ] Even though you've adjusted the viewport there still seems to be an overflow in your landing page I'd highly recommend you fix it as it leads to a bad UX. 😟
- Here's how it looks:
  ![firefox_bC7FdCccVI](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/104968236/de889a14-20fa-49ff-9a4b-262a34b2f982)
- [ ] Please make sure to check the overflow using a `375px` screen width on your browser.